content/docs/start/experiments.md

```diff
@@ -4,237 +4,227 @@ title: 'Get Started: Experiments'
+### 💡 Expand to see what this command does.
 
-The `-M` option here specifies a metrics file, while `--plots-no-cache`
-specifies a plots file produced by this stage that will not be
-<abbr>cached</abbr> by DVC. `dvc run` generates a new stage in the `dvc.yaml`
-file:
+`dvc exp run` is similar to `dvc repro` but with some added conveniences for
+running experiments. The `--set-param` (or `-S`) flag sets the values for
+[parameters](/doc/command-reference/params) as a shortcut to editing
+`params.yaml`.
 
-```yaml
-evaluate:
-  cmd: python src/evaluate.py model.pkl data/features ...
-  deps:
-    - data/features
-    - model.pkl
-    - src/evaluate.py
-  metrics:
-    - scores.json:
-        cache: false
-  plots:
-    - prc.json:
-        cache: false
-```
+Check that the `featurize.max_features` value has been updated in `params.yaml`:
 
-The biggest difference to previous stages in our pipeline is in two new
-sections: `metrics` and `plots`. These are used to mark certain files containing
-experiment "telemetry". Metrics files contain simple numeric values (e.g. `AUC`)
-and plots files contain matrices and data series (e.g. `ROC` or model loss
-plots) that are meant to be visualized and compared.
+```diff
+ featurize:
+-  max_features: 1500
++  max_features: 3000
+```
 
-> With `cache: false`, DVC skips caching the output, as we want `scores.json`
-> and `prc.json` to be versioned by Git.
+Any edits to dependencies (parameters, source code, data) will be reflected in
+the experiment run.
 
 </details>
 
-[`evaluate.py`](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py)
-writes the model's
-[AUC value](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
-to `scores.json`, which is marked as a metrics file with `-M`:
+`dvc exp diff` compares experiments:
 
-```json
-{ "auc": 0.57313829 }
-```
-
-It also writes `precision`, `recall`, and `thresholds` arrays (obtained using
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
-into plots file `prc.json`:
+```dvc
+$ dvc exp diff
+Path         Metric    Value    Change
+scores.json  avg_prec  0.56191  0.009322
+scores.json  roc_auc   0.93345  0.018087
 
-```json
-{
-  "prc": [
-    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...,
-    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
-  ]
-}
+Path         Param                   Value    Change
+params.yaml  featurize.max_features  3000     1500
 ```
 
-> DVC doesn't force you to use any specific file names, or even format or
-> structure of a metrics or plots file - it's pretty much user and case defined.
-> Please refer to `dvc metrics` and `dvc plots` for more details.
+## Iterating over experiments
 
-Let's save this experiment, so we can compare it later:
+The real magic of `dvc exp run` happens when running multiple experiments. So
```

Comment:
I changed this section to "Simultaneously running multiple experiments." Thoughts?

Reply:
> I changed this to "Simultaneously running multiple experiments."

It's more precise for sure. But it's not _really_ simultaneous unless you use `--jobs > 2` (and I'd go for "parallel" in that case).

How about something explicit like "Experiment queues"? Or if you prefer an action like in your other titles, maybe "Queueing experiment runs"? I guess most people should know what queueing is and be able to infer why it could be useful here.

Reply:
Yeah, "queueing" is a bit of an odd word, but this is an example where it fits well, and it's the term used in the command, so I'll go with that.